crates/fluvio-cli/src/consume/mod.rs

```diff
@@ -438,6 +447,11 @@ impl ConsumeOpt {
                                 &mut maybe_terminal_stdout,
                                 &mut maybe_table_model,
                             );
+
+                            if potential_offset == record.offset {
+                                println!("Ending offset has been reached; exiting consumer stream.");
```

Comment:
I think this should go to stderr
```suggestion
                                eprintln!("Ending offset has been reached; exiting consumer stream.");
```